contracts/gateway/EIP20CoGateway.sol

```diff
@@ -775,6 +775,10 @@ contract EIP20CoGateway is GatewayBase {
             _rlpParentNodes.length != 0,
             "RLP parent nodes must not be zero."
         );
+        require(
+            UtilityTokenInterface(utilityToken).exists(_beneficiary),
+            "Beneficiary address must be a registered address in utility token."
```

Comment:
We don't register addresses at the mosaic layer. Maybe, a more general message like 
`Beneficiary address must be allowed to hold utility token.`